static/nodejs_server/utils.ts

```diff
@@ -52,7 +58,49 @@ export function getProcessedSvg(chartSvg: SVGSVGElement): string {
     node.setAttribute("font-family", FONT_FAMILY);
     node.setAttribute("font-size", FONT_SIZE);
   });
-  // Get and return the svg as an xml string
+  // Create and return an image element out of the chart svg.
   const svgXml = xmlserializer.serializeToString(chartSvg);
-  return "data:image/svg+xml," + encodeURIComponent(svgXml);
+  const img = document.createElement("img");
+  img.src = "data:image/svg+xml," + encodeURIComponent(svgXml);
+  return img.outerHTML;
```

Comment:
Instead of returning an SVG wrapped in an image, why not return a proper SVG directly? Then the consumer could either use the SVG directly or wrap it in an image themselves if they wanted to

Reply:
done